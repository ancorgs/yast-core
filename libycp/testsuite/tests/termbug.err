Parsed:
----------------------------------------------------------------------
{
  term `contents = `VBox ();
  `contents = (x (contents (`VSpacing (1))));
  return contents;
}
----------------------------------------------------------------------
[Interpreter] tests/termbug.ycp:4 Undefined function contents()
[Interpreter] tests/termbug.ycp:4 Undefined function x()
Parsed:
----------------------------------------------------------------------
{
  term `a = ``(y2error ("hell"));
  return a;
}
----------------------------------------------------------------------
