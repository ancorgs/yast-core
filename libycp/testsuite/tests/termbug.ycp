// termbug
{
    term contents = `VBox ();
    contents = x (contents (`VSpacing (1)));
//                          ^ , is missing here

    return contents;
}

{
    term a=``(y2error("hell"));

    return a;
}
