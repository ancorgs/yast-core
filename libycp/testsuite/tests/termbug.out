(nil)
(y2error ("hell"))
